packages/vant/src/image-preview/README.md

```diff
@@ -302,7 +302,7 @@ imagePreviewRef.value?.swipeTo(1);
 | --- | --- | --- |
 | index | Custom index | _{ index: index of current image }_ |
 | cover | Custom content that covers the image preview | - |
-| image | Custom image content | _{ src: current image src }_ |
+| image | Custom image content | _{ src: current image src, onLoad: load image, style: current image style }_ |
```

Comment:
Maybe we can add an example to tell user when should they use the `style` and `onLoad`

Reply:
get

Reply:
I will merge this PR first, and feel free to open another PR for the example~

Reply:
Thanks. I've been feeling a bit unwell lately. It would be great if anyone is willing to complete this task.

Reply:
Take care of yourself ❤️

The documentation has been added: https://github.com/youzan/vant/pull/12751